cli/src/cli_util.rs

```diff
@@ -828,17 +829,18 @@ impl WorkspaceCommandEnvironment {
             workspace_name: &self.workspace_name,
         };
         let now = if let Some(timestamp) = self.settings.commit_timestamp() {
-            chrono::Local
-                .timestamp_millis_opt(timestamp.timestamp.0)
-                .unwrap()
+            jiff::Zoned::new(
+                Timestamp::from_millisecond(timestamp.timestamp.0).unwrap(),
+                TimeZone::system(),
+            )
```

Comment:
There is also [`Timestamp::to_zoned`](https://docs.rs/jiff/latest/jiff/struct.Timestamp.html#method.to_zoned), which might be a bit more succinct here.